src/Microsoft.NET.Workloads/Microsoft.NET.Workloads.csproj

```diff
@@ -25,15 +25,15 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Signed.WiX" Version="$(WixPackageVersion)" GeneratePathProperty="true" />
-    <PackageReference Include="MicroBuild.Plugins.SwixBuild.Dotnet" Version="$(SwixPackageVersion)" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Plugins.SwixBuild" Version="$(SwixPackageVersion)" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.Installers" Version="$(MicrosoftDotNetBuildTasksInstallersVersion)" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.Workloads" Version="$(MicrosoftDotNetBuildTasksWorkloadsPackageVersion)" GeneratePathProperty="true" />
   </ItemGroup>
 
   <PropertyGroup>
     <WixToolsetPath>$(PkgMicrosoft_Signed_Wix)\tools</WixToolsetPath>
-    <SwixPluginPath>$(PkgMicroBuild_Plugins_SwixBuild_Dotnet)</SwixPluginPath>
-    <SwixBuildTargets>$(SwixPluginPath)\build\MicroBuild.Plugins.SwixBuild.targets</SwixBuildTargets>
+    <SwixPluginPath>$(PkgMicrosoft_VisualStudioEng_MicroBuild_Plugins_SwixBuild)</SwixPluginPath>
+    <SwixBuildTargets>$(SwixPluginPath)\build\Microsoft.VisualStudioEng.MicroBuild.Plugins.SwixBuild.targets</SwixBuildTargets>
```

Comment:
Do we still need swix build targets if we have other logic to build the vsman? Or is this something else?

Reply:
This block of XML isn't related to the vsman. This is needed to create the MSI, but I don't know the full details since it is driven by the `CreateVisualStudioWorkloadSet` task that Jacques added to Arcade. I only had to update the block here since I'm using a much newer package version now, and they changed the package name from `MicroBuild.Plugins.SwixBuild` to `Microsoft.VisualStudioEng.MicroBuild.Plugins.SwixBuild`. I use that custom SDK out of this package that I showed you in the `.vsmanproj` file.